### PR TITLE
fix: fix media player supported features for HA beta

### PR DESCRIPTION
**Pull request by Petro31**
Fixes change in 2024.1 for supported features.

Fixes https://github.com/alandtse/alexa_media_player/issues/2149

### CODE REVIEW

File: custom_components/alexa_media/media_player.py [hunk 1]
```diff
@@ -20,21 +20,8 @@
...
-    SUPPORT_PREVIOUS_TRACK,
-    SUPPORT_SELECT_SOURCE,
-    SUPPORT_SHUFFLE_SET,
-    SUPPORT_STOP,
-    SUPPORT_TURN_OFF,
-    SUPPORT_TURN_ON,
-    SUPPORT_VOLUME_MUTE,
-    SUPPORT_VOLUME_SET,
-)
+from homeassistant.components.media_player.const import MEDIA_TYPE_MUSIC
```
Comment (larena1) [inline custom_components/alexa_media/media_player.py]: That is also deprecated

**Comment by MJHeijster** [comment]
Thanks, hope it gets merged soon!

**Comment by alandtse** [comment]
I guess we need to bumps hacs.json to set a new minimum version of HA.

**Review by alandtse — CHANGES_REQUESTED** [review]
Please bump this line to cover the new HA version 2024.1. https://github.com/alandtse/alexa_media_player/blob/dev/hacs.json#L6. Also, can you check on the comment about the deprecation of `MEDIA_TYPE_MUSIC`. Ideally we can handle them all at the same time.

**Comment by airy10** [comment]
Just confirming that it's fixing it for me, including the updating of sensors devices attached to a player

**Comment by Petro31** [comment]
FYI, I didn't change STATE_UNAVAILABLE.  That's supposed to be handled by the available property only, not in the state.  I'll let you decide what to do with that.

When the state property returns None, the state will be translated to `unknown`.  When the availability property returns False, the state will translate to `unavailable`.

**Comment by alandtse** [comment]
Thanks!

**Pull request merged**